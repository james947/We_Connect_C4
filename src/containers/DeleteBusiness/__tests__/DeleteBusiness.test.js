import { shallow } from 'enzyme'
import React from 'react';
import { DeleteBusiness } from '../';

describe("<DeleteBusiness />", () => {
    it('renders correctly', () => {
        const wrapper = shallow(<DeleteBusiness />);
        expect(wrapper).toMatchSnapshot();
    })
})