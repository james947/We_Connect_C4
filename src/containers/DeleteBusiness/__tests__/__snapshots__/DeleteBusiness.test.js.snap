// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DeleteBusiness /> renders correctly 1`] = `
<div>
  <button
    class="btn btn-primary"
    data-target="#exampleModalCenter"
    data-toggle="modal"
    type="button"
  >
    Delete Business
  </button>
  <div
    aria-hidden="true"
    aria-labelledby="exampleModalCenterTitle"
    class="modal fade"
    id="exampleModalCenter"
    role="dialog"
    tabindex="-1"
  >
    <div
      class="modal-dialog modal-dialog-centered"
      role="document"
    >
      <div
        class="modal-content"
      >
        <div
          class="modal-header"
        >
          <h5
            class="modal-title"
            id="exampleModalLongTitle"
          >
            Confirm Delete
          </h5>
          <button
            aria-label="Close"
            class="close"
            data-dismiss="modal"
            type="button"
          >
            <span
              aria-hidden="true"
            >
              Ã—
            </span>
          </button>
        </div>
        <div
          class="modal-body"
        >
          <h4>
            Are you sure you want to delete this business?
          </h4>
        </div>
        <div
          class="modal-footer"
        >
          <button
            class="btn btn-secondary"
            data-dismiss="modal"
            type="button"
          >
            Close
          </button>
          <button
            class="btn btn-danger"
            onClick={[Function]}
            type="button"
          >
            Delete
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
